lib/gpu_memory_service/tests/_deps.py

```
@@ -29,12 +29,20 @@ def _check_gms_usable() -> bool:
 HAS_GMS = _check_gms_usable()
 
-# CUDA availability requires a full torch import
+# CUDA / XPU availability requires a full torch import
 HAS_CUDA = False
+HAS_XPU = False
 if HAS_TORCH:
     import torch
 
     try:
         HAS_CUDA = torch.cuda.is_available()
     except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)